pkg/controllers/allocation/scheduling/scheduler.go

```diff
@@ -102,13 +102,19 @@ func globalRequirements(instanceTypes []cloudprovider.InstanceType) (requirement
 		v1.LabelInstanceTypeStable: sets.NewString(),
 		v1.LabelTopologyZone:       sets.NewString(),
 		v1.LabelArchStable:         sets.NewString(),
-		v1.LabelOSStable:           sets.NewString(),
+		v1alpha5.LabelCapacityType: sets.NewString(),
 	}
 	for _, instanceType := range instanceTypes {
+		scheduledZoneSet := sets.String{}
+		scheduledCapacityTypeSet := sets.String{}
```

Comment:
I'd suggest removing `Set` from the variable names.

Reply:
It's a personal habit of mine from Java, I like having the type in the variable names (especially in code chunks where I use the same variable prefix because it's different forms of a singular entity), but I can stop doing that here.